bciers/apps/reporting/src/app/components/operations/Operations.tsx

```diff
@@ -0,0 +1,51 @@
+import { GridRowsProp } from "@mui/x-data-grid";
+
+import { actionHandler } from "@/app/utils/actions";
+import { getServerSession } from "next-auth";
+import { authOptions } from "@/app/api/auth/[...nextauth]/route";
+import { OperationRow } from "@reporting/src/app/components/operations/types";
+import ReportingOperationDataGrid from "./ReportingOperationDataGrid";
+
+export const formatOperationRows = (rows: GridRowsProp) => {
+  return rows.map(({ id, operator, name, bcghg_id }) => {
+    return {
+      id,
+      name,
+      bcghg_id,
+      operator: operator,
+    };
```

Comment:
Just for consistency in the coding style

```suggestion
    return {
      id,
      name,
      bcghg_id,
      operator,
    };
```

Reply:
Hey! Sorry just poked my head in here after the demo, hope this was relatively easy to implement from our examples :)

 `formatOperationRows` can be removed altogether, if you notice it's not doing anything (map returning the same values). It's some registration 1 tech debt that we removed in the recent Registration part 2 Operations table.